sp/src/vscript/squirrel/include/sqstdtime.h

```diff
@@ -0,0 +1,88 @@
+//-----------------------------------------------------------------------
```

Comment:
Should ideally keep squirrel as being unmodified

Reply:
Modifying how, do you think the file should be located elsewhere?

Reply:
This file is stored in the squirrel folder which is currently just a direct copy of squirrel with no changes, adding this file in that folder changea that